src/id-verification/Camera.jsx

```
@@ -26,6 +32,83 @@ class Camera extends React.Component {
     this.cameraPhoto.stopCamera();
   }
 
+  startDetection() {
+    this.setState({ shouldDetect: !this.state.shouldDetect });
+    while (!this.state.hasLoaded) {
+      // wait until video has loaded
+    }
+    const loadModelPromise = blazeface.load();
+    Promise.all([loadModelPromise])
+      .then((values) => {
+        this.detectFromVideoFrame(values[0], this.videoRef.current);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }
+
+  detectFromVideoFrame = (model, video) => {
+    model.estimateFaces(video).then((predictions) => {
```

This line causes an error when you reach the next page, I think because this function is still running for a short period of time after the video.ref is gone. I had originally added try/catch logic to ignore this error, but I feel like there's a better way to handle this.